py4DSTEM/process/diffraction/crystal_ACOM.py

```diff
@@ -1428,160 +1428,108 @@ def match_single_pattern(
         return orientation
 
 
+def orientation_map_to_orix_CrystalMap(self, 
+                                       orientation_map, 
+                                       ind_orientation=0, 
+                                       pixel_size=1.0, 
+                                       pixel_units='px', 
+                                       return_color_key=False
+                                       ):
+    from orix.quaternion import Rotation, Orientation
+    from orix.crystal_map import CrystalMap, Phase, PhaseList, create_coordinate_arrays
+    from orix.plot import IPFColorKeyTSL
+    from diffpy.structure import Atom, Lattice, Structure
+    
+    from pymatgen.symmetry.analyzer import SpacegroupAnalyzer
+    from pymatgen.core.structure import Structure as pgStructure
+    
+    from py4DSTEM.process.diffraction.utils import element_symbols
```

Comment:
Trivial, but you also create the `elements` dict in `utils.py` which you then don't use here, or anywhere else as far as I can see?

Reply:
it’s just a nice thing to have :)